addons/html_editor/static/src/fields/html_field.js

```diff
@@ -188,6 +206,9 @@ export const htmlField = {
         return {
             editorConfig,
             isCollaborative: options.collaborative,
+            dynamicPlaceholder: options["dynamic_placeholder"],
+            dynamicPlaceholderModelReferenceField:
+                options["dynamic_placeholder_model_reference_field"],
```

Comment:
To keep the style a bit more consistent:
```js
            dynamicPlaceholder: options.dynamic_placeholder,
            dynamicPlaceholderModelReferenceField:
                options.dynamic_placeholder_model_reference_field,
```

Reply:
:ok_hand: